discovery_engine_core/web-api2/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "web-api2"
+version = { workspace = true }
+edition = { workspace = true }
+rust-version = { workspace = true }
+description = "Wep Api server for the document personalization service."
+license = "AGPL-3.0-only"
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+actix-web = "4.2.1"
```

Comment:
:art: actix enables quite a lot default features, maybe we can enable only some required features to save space and time.